spec/shared/models/floating_ip.rb

```diff
@@ -0,0 +1,9 @@
+shared_examples :floating_ip do |provider_name|
```

Comment:
Yes, hopefully all provider gems that implement/subclass FloatingIp like https://github.com/ManageIQ/manageiq-providers-amazon/pull/49